.travis.yml

```diff
@@ -65,19 +65,19 @@ matrix:
               else curl --progress-bar --upload-file openrct2-linux.tar.gz https://transfer.sh/openrct2-linux-i686.tar.gz -o link && cat link;
               fi
         - os: linux
-          env: OPENRCT2_CMAKE_OPTS="-G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_SHARED_LIBS=ON -DDISABLE_RCT2_TESTS=on" TARGET=ubuntu_amd64
+          env: OPENRCT2_CMAKE_OPTS="-G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_SHARED_LIBS=ON -DENABLE_RCT2_TESTS=NO" TARGET=ubuntu_amd64
         - os: linux
-          env: OPENRCT2_CMAKE_OPTS="-G Ninja -DCMAKE_TOOLCHAIN_FILE=../CMakeLists_mingw.txt -DFORCE32=on -DBUILD_SHARED_LIBS=ON -DDISABLE_RCT2_TESTS=on" TARGET=windows
+          env: OPENRCT2_CMAKE_OPTS="-G Ninja -C../cmake/configs/LinuxMinGW.cmake -DBUILD_SHARED_LIBS=ON -DENABLE_RCT2_TESTS=NO" TARGET=windows
```

Comment:
is `-C` a shorthand for `-DCMAKE_TOOLCHAIN_FILE=`?

Reply:
`LinuxMinGW` implies that Linux is required to build it, which is not true. Mingw-w64 can be used from other systems, too.

FWIW: We require Mingw-w64, MinGW is a bit different thing. Their naming is tricky and there is some drama involved.

Reply:
`-C` loads up a file that sets initial cache values, so you don't have to specify a ton of `-D` options on the command line. You can give it a good, useful name and move the options for a particular build configuration into that single file.